validate.mk

```diff
@@ -2,4 +2,5 @@
 MM_SHELL ?= bash -c
 
 validate:
+	type mm.py || sudo pip install mechanical_markdown
```

Comment:
Probably move this to a new make-command, like `ensure-mm` - that installs mechanical markdown if needed.